docs/be/version_be.properties

```diff
@@ -1,2 +1,2 @@
 # NOI18N
-version=4.3.1
+version=4.2.0
```

Comment:
something degraded?

Reply:
This was modified in 290b6672

Reply:
@brandelune  is it intended change?

Reply:
Yes. The version that appears in the manual is always 4.2. There was no 4.3.1 version.

Reply:
A commit message did not said fixes you explains here, so looks doubt...

Reply:
Sorry. I wanted to make sure that the online manual version numbers did not confuse the users. This situation will be fixed as more manuals move to 6.0.